### PR TITLE
Fix travis build matrix for net-ssh >= 3.0

### PR DESCRIPTION
**Pull request by ydkn**
net-ssh >= 3.0 requires ruby >= 2.0.
jruby < 9.0.0.0 and ruby < 2.0 needs to be tested with net-ssh < 3.0.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
 rvm:
-- 2.0.0
-- jruby-19mode
-- rbx-2
-script: bundle exec rspec
+  - ruby-1.9.3
+  - ruby-2.0.0
+  - ruby-2.1.7
+  - ruby-2.2.3
```
Comment (afeld) [inline .travis.yml:7]: Probably just 1.9.3 and 2.2.3 are fine.

Comment (afeld) [inline .travis.yml:7]: ...unless you have a specific reason for wanting to include those other versions?

**Comment by afeld** [comment]
Just one little note. Otherwise, looks great!

File: .travis.yml (hunk 1 above)
Comment (ydkn) [inline .travis.yml:7]: didn't see your comment in time - but I think if it can run with RBX and jRuby -> why not test it.

**Comment by ydkn** [comment]
Sorry got a bit confused... (because I already merge this into the other PR - which in hindsight was not the best idea)

No specific reason for also including 2.0.0 and 2.1.7... will remove them in a moment.

**Pull request merged**